### DISCUSSION TITLE
api设置dns通用参数提示blocking_mode: incorrect value

### DISCUSSION THREAD

**Discussion by steamsv** [AdguardTeam/AdGuardHome#3469] [Q&A] [answered]
curl http://ip:3000/control/dns_config -H 'accept: */*' -H 'Authorization: Basic xxxx' -H 'Content-Type: application/json' -d '{"blocking_mode": "custom_ip","blocking_ipv4": "1.2.3.4"}'

以上示例
使用default没有问题

**Comment by ainar-g** [comment, marked as answer]
Hello.  When using the Custom IP blocking mode, both IPv4 and IPv6 addresses must be provided.  Please try:

```json
{"blocking_mode":"custom_ip","blocking_ipv4":"1.2.3.4","blocking_ipv6":"1234::cdef"}
```

**Reply by steamsv** [reply]
Thank you, successful setting
